docker/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Copyright 2022, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+case "$(uname -m)" in
+    arm64)
+        DOCKER="docker buildx build --platform linux/amd64"
+        ;;
+    *)
+        DOCKER="docker build"
+        ;;
+esac
```

Comment:
An alternative, if buildx is generally available, is _always_ to use buildx and platform --- so whatever you're running on it uses the right platform.  Although I've noticed the system docker for debian unstable does not include it.

Reply:
`buildx` is still relatively new and unfortunately not generally available (e.g. Debian default)